### ISSUE TITLE
WakeupException not thrown for partitioned consumer

### ISSUE THREAD

**Issue by yarosman** [akka/alpakka-kafka#329]
Hello.
In case of creating consumer for specified partitioned topic with invalid kafka host, I don't receive any WakeupException, and stream not completed
```
val control = Consumer.plainSource(consumerSettings, Subscriptions.assignmentWithOffset(new TopicPartition(topic, 0) -> offset))
          .mapAsync(1)(msg => offsetStore.save(msg.offset()))
          .to(Sink.ignore)
          .run()
```

But in case for Subscriptions.topics(topic), WakeupException occurred several time and stream closed

```
val control = Consumer.plainSource(consumerSettings, Subscriptions.topics(topic))
          .mapAsync(1)(msg => offsetStore.save(msg.offset()))
          .to(Sink.ignore)
          .run()
```

So how to solve this ?

**Comment by patriknw** [comment]
I can confirm that I also see this. `consumer.poll` returns immediately in this case.
I see logs:
```
WARN  o.apache.kafka.clients.NetworkClient - Connection to node -1 could not be established. Broker may not be available.
```

and when starting broker again:
```
WARN  o.apache.kafka.clients.NetworkClient - Error while fetching metadata with correlation id 1 : {topic1-bfed0ebf-2c6f-4d5c-ad6d-959f2ba3e297=LEADER_NOT_AVAILABLE}

org.apache.kafka.common.errors.NotLeaderForPartitionException: This server is not the leader for that topic-partition.
```

I don't know what to do about it.

**Comment by jroper** [comment]
Not sure if related, but I sporadically see the same behaviour for `Subscriptions.topics`.

**Comment by ennru** [comment]
Fixed with #617
